exporter/datadogexporter/config.go

```diff
@@ -242,6 +242,20 @@ type TracesConfig struct {
 	flushInterval float64
 }
 
+// LogsConfig defines logs exporter specific configuration
+type LogsConfig struct {
+	// TCPAddr.Endpoint is the host of the Datadog intake server to send logs to.
+	// If unset, the value is obtained from the Site.
+	confignet.TCPAddr `mapstructure:",squash"`
+
+	// SendLogRecordBody when enabled would send string representation of body field of Otel log record as message to Datadog
```

Comment:
This one is better but still needs to be improved. Let me follow up with another comment after our discussion.

Reply:
we have removed the config